src/diffusers/models/autoencoder_kl.py

```diff
@@ -158,6 +189,108 @@ def decode(self, z: torch.FloatTensor, return_dict: bool = True) -> Union[Decode
 
         return DecoderOutput(sample=decoded)
 
+    def blend_v(self, a, b, blend_width):
```

Comment:
Would avoiding the use of `width` in both these methods prevent confusion? Not convinced about `blend_size` though. Maybe something like `blend_extent`? If you think it doesn't help then it's ok.

Reply:
Sounds good!